extensions/mssql/src/localizedConstants.ts

```diff
@@ -55,3 +55,10 @@ export function sparkJobSubmissionGetApplicationIdFailed(err: string): string {
 export function sparkJobSubmissionLocalFileNotExisted(path: string): string { return localize('sparkJobSubmission.LocalFileNotExisted', "Local file {0} does not existed. ", path); }
 export const sparkJobSubmissionNoSqlBigDataClusterFound = localize('sparkJobSubmission.NoSqlBigDataClusterFound', "No SQL Server Big Data Cluster found.");
 export function sparkConnectionRequired(name: string): string { return localize('sparkConnectionRequired', "Please connect to the Spark cluster before View {0} History.", name); }
+
+
+export const failedToFindTenants = localize('mssql.failedToFindTenants', "Failed to find tenants");
+export const tokenRefreshFailed = localize('mssql.tokenRefreshFailed', "Editor token refresh failed");
```

Comment:
could you please refine the variable name and the messages?

Reply:
as a user, I am more interested in what I should do in case of error.

Reply:
Agreed. "Editor token" means nothing to a user. What token? Please be as specific as possible.

(be careful here of this logic not necessarily being used for intellisense. You may need to have the request pass in some sort of context name so that this can display a generic error message with that name instead of assuming it's always Intellisense.